### DISCUSSION TITLE
Unauth APIs

### DISCUSSION THREAD

**Discussion by ddugovic** [gbtami/pychess-variants#978] [Ideas]
I'd make a PyChess Statbot like https://top.gg/bot/490949867657494530 but there are no unauthenticated APIs to target.

**Comment by gbtami** [comment]
We can add a Statbot to the built in users list similar to Discord-Relay here:
https://github.com/gbtami/pychess-variants/blob/master/server/server.py#L128
Feel free to implement the missing APIs you need in a PR.
